docs/README.md

```diff
@@ -124,7 +124,7 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `productAssemblyGroupNameRow` |
 | `productAssemblyGroupRequiredTag` |
 | `productAssemblyOptionItemCustomize` |
-| `productAssemblyOptionItemCustomize__label` |
+| `productAssemblyOptionItemCustomizeLabel` |
```

Comment:
This is a breaking change. Stores could be using the `productAssemblyOptionItemCustomize__label`.

Reply:
@victorhmp actually it's not! There's something weird with this branch: it doesn't share the same history with master, but it is letting us merge 🤔 

I've fixed this handle [on this PR](https://github.com/vtex-apps/product-customizer/pull/80). It was preventing the app from building (other previous was merged incorrectly).

What I suggest @gustavopvasconcellos to do is to rebase your branch on the vtex-apps/master branch. That way, this fix will no longer be necessary.

Reply:
Awesome!! Nice catch @icazevedo 😄